docs/topics/feed-exports.rst

```diff
@@ -515,6 +515,11 @@ which uses safe numeric encoding (``\uXXXX`` sequences) for historic reasons.
 
 Use ``utf-8`` if you want UTF-8 for JSON too.
 
+.. versionchanged:: 2.8
```

Comment:
2.8 here should be fine but please use `VERSION` next time to be more flexible.

Reply:
```diff
-.. versionchanged:: 2.8
+.. versionchanged:: VERSION
```
Like this?

Reply:
Yes, it will be replaced during the release process.

Reply:
Cool, done as well.